module_3/gifts/controller/src/main/resources/messages_ru.properties

```diff
@@ -0,0 +1,48 @@
+exception.100000=\u041D\u0435\u043A\u043E\u0442\u043E\u0440\u044B\u0435 \u043F\u043E\u043B\u044F \u0441\u0435\u0440\u0442\u0438\u0444\u0438\u043A\u0430\u0442\u0430 \u043D\u0435\u0432\u0430\u043B\u0438\u0434\u043D\u044B, \u043F\u043E\u0436\u0430\u043B\u0443\u0439\u0441\u0442\u0430, \u0438\u0441\u043F\u0440\u0430\u0432\u044C\u0442\u0435 \u043E\u0448\u0438\u0431\u043A\u0438
```

Comment:
@LizavetaSinitsyna In the future try to create a new file in UTF-8 encoding later. This allows you to see it in Russian.